frontend/src/components/NewRunParametersV2.tsx

```diff
@@ -137,7 +149,11 @@ function NewRunParametersV2(props: NewRunParametersProps) {
                   nextUpdatedParameters[k] = { string_value: value };
                   setUpdatedParameters(nextUpdatedParameters);
                   if (props.handleParameterChange) {
-                    props.handleParameterChange(nextUpdatedParameters);
```

Comment:
Please add test to validate your changes in https://github.com/kubeflow/pipelines/blob/master/frontend/src/components/NewRunParametersV2.test.tsx